samples/zigbee/light_bulb/src/zcl_scenes.c

```diff
@@ -0,0 +1,939 @@
+		LOG_INF("Save level control state inside scene table");
+		entry->level_control.has_current_level = ZB_TRUE;
+	}
+	if (has_cluster(ZB_ZCL_CLUSTER_ID_WINDOW_COVERING)) {
+		LOG_INF("Save window covering state inside scene table");
+		if (get_current_lift_value(
+		    &entry->window_covering.current_position_lift_percentage) == RET_OK) {
+			entry->window_covering.has_current_position_lift_percentage = ZB_TRUE;
+		}
+		if (get_current_tilt_value(
+		    &entry->window_covering.current_position_tilt_percentage) == RET_OK) {
+			entry->window_covering.has_current_position_tilt_percentage = ZB_TRUE;
+		}
+	}
+}
+
+static void recall_scene(struct scene_table_on_off_entry *entry)
+{
+	zb_bufid_t buf = zb_buf_get_any();
+	zb_zcl_attr_t *attr_desc;
+	zb_ret_t result;
+
+	if (entry->on_off.has_on_off) {
+		LOG_INF("Recall On/Off state");
+
+		attr_desc = zb_zcl_get_attr_desc_a(
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_ON_OFF,
+			ZB_ZCL_CLUSTER_SERVER_ROLE,
+			ZB_ZCL_ATTR_ON_OFF_ON_OFF_ID);
+
+		ZB_ZCL_INVOKE_USER_APP_SET_ATTR_WITH_RESULT(
+			buf,
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_ON_OFF,
+			attr_desc,
+			&entry->on_off.on_off,
+			result
+		);
+	}
+	if (entry->level_control.has_current_level) {
+		LOG_INF("Recall level control state");
+
+		attr_desc = zb_zcl_get_attr_desc_a(
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_LEVEL_CONTROL,
+			ZB_ZCL_CLUSTER_SERVER_ROLE,
+			ZB_ZCL_ATTR_LEVEL_CONTROL_CURRENT_LEVEL_ID);
+
+		ZB_ZCL_INVOKE_USER_APP_SET_ATTR_WITH_RESULT(
+			buf,
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_LEVEL_CONTROL,
+			attr_desc,
+			&entry->level_control.current_level,
+			result
+		);
+	}
+	if (entry->window_covering.has_current_position_lift_percentage) {
+		LOG_INF("Recall window covering lift state");
+
+		attr_desc = zb_zcl_get_attr_desc_a(
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_WINDOW_COVERING,
+			ZB_ZCL_CLUSTER_SERVER_ROLE,
+			ZB_ZCL_ATTR_WINDOW_COVERING_CURRENT_POSITION_LIFT_PERCENTAGE_ID);
+
+		ZB_ZCL_INVOKE_USER_APP_SET_ATTR_WITH_RESULT(
+				buf,
+				ZCL_SCENES_ENDPOINT,
+				ZB_ZCL_CLUSTER_ID_WINDOW_COVERING,
+				attr_desc,
+				&entry->window_covering.current_position_lift_percentage,
+				result
+		);
+	}
+	if (entry->window_covering.has_current_position_tilt_percentage) {
+		LOG_INF("Recall window covering tilt state");
+
+		attr_desc = zb_zcl_get_attr_desc_a(
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_WINDOW_COVERING,
+			ZB_ZCL_CLUSTER_SERVER_ROLE,
+			ZB_ZCL_ATTR_WINDOW_COVERING_CURRENT_POSITION_TILT_PERCENTAGE_ID);
+
+		ZB_ZCL_INVOKE_USER_APP_SET_ATTR_WITH_RESULT(
+			buf,
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_WINDOW_COVERING,
+			attr_desc,
+			&entry->window_covering.current_position_tilt_percentage,
+			result
+		);
+	}
+
+	zb_buf_free(buf);
+}
+
+static void send_view_scene_resp(zb_bufid_t bufid, zb_uint16_t idx)
+{
+	zb_uint8_t *payload_ptr;
+	zb_uint8_t view_scene_status = ZB_ZCL_STATUS_NOT_FOUND;
+
+	LOG_INF(">> %s bufid %hd idx %d", __func__, bufid, idx);
```

Comment:
Wouldn't it be better to have these entry/exit logs at debug level? INF is the default log level and it looks that these logs could obscure the output a bit.

Reply:
Entry-to-function and exit logs changed from INF to DBG